forcepointwebsecurity_connector.py

```diff
@@ -406,10 +406,10 @@ def _lookup_objects(self, action_result, objects):
             for curr_obj_type in ['IPs', 'URLs']:
                 for curr_obj in cat_details[curr_obj_type]:
                     # Because URLs may or may not include the protocol from the input parameter, check if either match
-                    if curr_obj in category_map.keys():
+                    if curr_obj in list(category_map.keys()):
```

Comment:
When checking if a key is in a dict with py3, you can use `if obj in dict:` instead of `if obj in list(dict.keys())`

Reply:
Done